packages/hash/frontend/src/blocks/page/BlockContextMenu/LoadEntityMenuContent.tsx

```diff
@@ -86,6 +86,19 @@ export const LoadEntityMenuContent: FunctionComponent<
       // – block entities are simply a reference to (a) a component and (b) a child entity
       if (isBlockEntity(entity)) return false;
 
+      /**
+       * loading text entities does not work, possibly due to use of legacy __linkedData
+       * @todo see if this works when __linkedData is removed
+       */
+      if (
+        isTextEntity(entity) ||
+        ["Page", "Paragraph", "Header", "Callout"].includes(
+          entity.entityType.properties.title,
+        )
```

Comment:
You're probably better off using `isTextContainingEntityProperties` than using the entity type name for Paragraph, Header, Callout – it's more consistent and it means we won't forget about this when reforming text blocks

```suggestion
        isTextEntity(entity) ||
        isTextContainingEntityProperties(entity.properties) ||
        entity.entityType.properties.title === "Page"
```

I'd love a better way to work out if its a page entity – perhaps you can fetch the entity type id for the page system type or something, but then this will likely change when we move to the HASH api anyway

Reply:
https://github.com/hashintel/hash/pull/1014/commits/0f6b38a572f46b6190083c4edca2231134a1383f